db/migrations/create_db_tables.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE users (
+  id SERIAL PRIMARY KEY,
+  email VARCHAR(60) UNIQUE,
+  password VARCHAR(140),
+  name VARCHAR(200),
+  username VARCHAR(100) UNIQUE
+);
+
+CREATE TABLE messages (
+  id SERIAL PRIMARY KEY,
+  content VARCHAR(2000),
+  user_id INTEGER REFERENCES users(id) ON DELETE CASCADE,
+  created_at TIMESTAMP DEFAULT CURRENT_TIMESTAMP::TIMESTAMP(0)
```

Comment:
Great to know there's a TIMESTAMP data type in SQL! I had assumed you're have to use Ruby's Time class to create this information.